Description: directfb ftbfs on armel. Implicit declaration error
 Fix for ARM builds failing on:
  ../../../gfxdrivers/davinci/davinci_c64x.c: In function 'davinci_c64x_open':
  ../../../gfxdrivers/davinci/davinci_c64x.c:1900:6: error: implicit declaration of function 'mknod' [-Werror=implicit-function-declaration]
  cc1: some warnings being treated as errors
 .
 directfb (1.2.10.0-4.3) unstable; urgency=low
 .
   * Non-maintainer upload.
   * Fix "directfb ftbfs on armel. Implicit declaration error"
     - Thanks Peter Green for bug report and patch (Closes: #644782)
   * Remove m4 macros when calling clean target
Author: Hector Oron <zumbi@debian.org>
Bug-Debian: http://bugs.debian.org/644782

--- directfb-1.2.10.0.orig/gfxdrivers/davinci/davinci_c64x.c
+++ directfb-1.2.10.0/gfxdrivers/davinci/davinci_c64x.c
@@ -37,6 +37,8 @@
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/types.h>
+#include <sys/stat.h>
+
 #include <unistd.h>
 
 #include <directfb_util.h>
